tests/time_tests/test_runner/test_timetest.py

```diff
@@ -41,6 +41,11 @@ def test_timetest(instance, executable, niter):
         "device": instance["device"]["name"],
         "niter": niter
     }
+    if exe_args["device"] == "GPU":
+        # Generate cl_cache via additional timetest run
+        _exe_args = exe_args.copy()
+        _exe_args["niter"] = 1
+        run_timetest(_exe_args, log=logging)
```

Comment:
please add a log_info before the "heat up" run explaining that this run is extra one

Reply:
Added. Also added assert to check cl_cache was generated